src/django/api/views/v1/opensearch_query_builder/opensearch_query_builder.py

```diff
@@ -41,8 +41,13 @@ def add_multi_match(self, query):
     def add_terms(self, field, values):
         if not values:
             return self.query_body
-        terms_field = f'{field}.alpha_2' if field == 'country' \
-            else f'{field}.keyword'
+
+        if (field == 'country'):
+            terms_field = f'{field}.alpha_2'
+        elif (field == 'os_id'):
+            terms_field = field
+        else:
+            terms_field = f'{field}.keyword'
```

Comment:
Please note that this set of if statements is starting to grow. If it continues to expand, it might become problematic to maintain. We need to consider a more extendable solution in the long run. There's nothing to do for now, but it's something we need to keep in mind the next time we try to extend this code.

Reply:
@VadimKovalenkoSNF There is two useful solution:
- Using **match -> case**
- Pre-define options for each case
```
build_options = {
  'country': build_country
  'os_id': build_os_id
}

action = build_options.get(field)
```